mod_reforged/hooks/skills/racial/schrat_racial.nut

```diff
@@ -1,4 +1,46 @@
 ::mods_hookExactClass("skills/racial/schrat_racial", function(o) {
+	local create = o.create;
+	o.create()
+	{
+		create();
+		this.m.Name = "Race: Schrat";
+		this.m.IsHidden = false;
+	}
+
+	o.getTooltip = function()	// This function doesn't exist yet in the vanilla effect, only in its parent
+	{
+		return [
+			{
+				id = 1,
+				type = "title",
+				text = this.getName()
+			},
+			{
+				id = 2,
+				type = "description",
+				text = this.getDescription()
+			},
+			{
+				id = 10,
+				type = "text",
+                icon = "ui/icons/melee_defense.png",
+				text = "Reduces damage taken from melee piercing attacks by " + ::MSU.Text.colorRed("50%")
```

Comment:
I'd reword this as `50% reduced melee piercing damage received`. Similarly for other lines like this.

Reply:
Do you have any reference for that or just now brainstorming on the fly about a better phrasing?

**Nitpicks on your formulation are:**
- `Reduces` and `received` are more similar than the `Reduces` and `taken`
- all the different possible damage types are squished together with `damage`. That makes it harder to parse than if there was some separating word between them.

I just looked up vanilla indomitable and there they use yet another variation:

> text = "Only receive [color=" + this.Const.UI.Color.PositiveValue + "]50%[/color] of any damage"

Berserker Potion uses:

> text = "[color=" + this.Const.UI.Color.PositiveValue + "]+" + (100 - this.Math.maxf(0.3, 1.0 - 0.02 * this.m.RageStacks) * 100) + "%[/color] damage reduction"

Reply:
> `Reduces and received are more similar than the Reduces and taken`

`Taken` is technically correct but `Received` is the more appropriate word for this case - as the opposite we use is `dealt` and not `given`. There is enough of a distance, in words, between `Reduced` and `received` that both being similar is not a problem.

> all the different possible damage types are squished together with damage.

I could not understand this part.

Reply:
My variant:

> Reduces **damage** taken from **melee piercing attacks** by 50%

Your variant:

> 50% reduced **melee piercing attack damage** received

> 50% reduced **melee piercing damage** received from **attacks**

Your first variant has most of the important information in one big pile together. 
Mine has a few filler words in between that may help the reader to better separate them in their head.
Especially if I was to also make **50%** and **reduced** bold which are also part of those important information

Reply:
I see. I think that's not a big deal.

1. 50% reduced melee piercing damage received
2. 50% reduced damage received from melee piercing attacks

1 is (a) more concise, (b) sufficient, (c) closer in analogy to `+x Damage`, so the tooltips language flow remains consistent.
2 is unnecessarily long - and when you have many of them stacked up in a tooltip, it bloats it with unnecessary words.

Having different styles across tooltips is inelegant design. We should have one consistent design.
`-5 damage received` is in the same flow as `5% reduced damage received` and not `damage received is reduced by 5%`.

The bigger deal for me is `Reduces` which should definitely be changed into something which is a description of "what happens" rather than which makes the effect a subject which does things.

Reply:
But where is the mention of `attacks` in your variant (1)?
Of course variant 2 is "unnecessary long" because otherwise it would be incorrect